src/pykeen/triples/triples_factory.py

```diff
@@ -284,11 +284,11 @@ def num_triples(self) -> int:  # noqa: D401
 
     def get_entity_ids(self) -> Collection[int]:
         """Get the set of entity identifiers."""
-        raise NotImplementedError  # TODO @mberr should these be pre-cached on __init__?
+        return list(range(self.num_entities))
```

Comment:
@mberr is this guaranteed?

Reply:
maybe we should just implement in the `CoreTriplesFactory.__init__` code that extracts the unique entities/relations. Those functions are probably even written somewhere already.

Reply:
Solved in abfceb0d